docs/milestone2_walkthrough.md

```diff
@@ -0,0 +1,168 @@
+# Milestone 2 Walkthrough
+
+## Walkthrough Goals
+
+* Get a local environment running with 1 control plane cluster and 2 workload clusters
+* Define a Gateway in the control plane with a HTTPS listener, and have it instantiated in 2 workload clusters
+* Deploy an Application with a HTTPRoute to the 2 workload clusters, and have the route attached to the Gateway listener
+* Curl the Application host with DNS resolving from Route 53 and a TLS Certificate
+* Apply a Kuadrant RateLimitPolicy to the Application
+* Curl the Application host and see the rate limit being enforced
+
+## Prerequisites
+
+* [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+* [AWS Account](https://aws.amazon.com/) with [Route53](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/Welcome.html) Zone permissions
+    * Choose an existing Hosted zone or create a new one for the walkthrough.
+* openssl>=3
+    * On macos a later version is available with `brew install openssl`. You'll need to update your PATH as macos provides an older version via libressl as well
+    * On fedora use `dnf install openssl`
+
+## Environment Setup
+
+Create a set of environment files with your AWS configuration.
+First, create a file called `aws-credentials.env` in the root of the project and add the following config, replacing values as necessary
+
+```bash
+AWS_ACCESS_KEY_ID=<AWS_ACCESS_KEY_ID>
+AWS_SECRET_ACCESS_KEY=<AWS_SECRET_ACCESS_KEY>
+AWS_REGION=<AWS_REGION>
+```
+
+Next, create a file called `controller-config.env` in the root of the project and add the following config, replacing values as necessary. For example, if your Hosted zone is for the domain `apps.example.com`, use that as the `ZONE_ROOT_DOMAIN`. This will mean your Application can use a hostname of `myapp.apps.example.com`
+
+```bash
+ZONE_ROOT_DOMAIN=<my.hosted.zone.domain.name>
+AWS_DNS_PUBLIC_ZONE_ID=<my.hosted.zone.id>
+```
+
+Start the local clusters and run the controller in the control plane
+
+```bash
+MCTC_WORKLOAD_CLUSTERS_COUNT=2 make local-setup
```

Comment:
i think it might be useful to split these 2 commands so if you want to just stop and start the operator you can, i have by accident a few times from memory muscle hit up to bring the last command thinking it will just start the operator forgetting the make local setup is apart of it and have to wait for the whole thing to go away and come again